integration/cli/build.configequality.test.ts

```diff
@@ -76,95 +74,104 @@ describe("build", () => {
-          kind.ServiceAccount,
-          kind.Secret,
-          kind.Service,
-          kind.Role,
-          kind.Deployment,
-          kind.MutatingWebhookConfiguration,
-          kind.ValidatingWebhookConfiguration,
-        ])("should have %s resources in both pepr and helm manifests", resourceKind => {
-          for (const { getResources } of sources) {
-            const res = resource.select(getResources(), resourceKind, peprUuid);
+        const resourceKinds = [
+          { kind: kind.ClusterRole, name: (uuid: string): string => uuid },
+          { kind: kind.ServiceAccount, name: (uuid: string): string => uuid },
+          { kind: kind.Secret, name: (uuid: string): string => `${uuid}-api-path` },
+          { kind: kind.Secret, name: (uuid: string): string => `${uuid}-tls` },
+          { kind: kind.Secret, name: (uuid: string): string => `${uuid}-module` },
+          { kind: kind.Service, name: (uuid: string): string => uuid },
+          { kind: kind.Service, name: (uuid: string): string => `${uuid}-watcher` },
+          { kind: kind.Role, name: (uuid: string): string => `${uuid}-store` },
+          { kind: kind.Deployment, name: (uuid: string): string => uuid },
+          { kind: kind.Deployment, name: (uuid: string): string => `${uuid}-watcher` },
+          { kind: kind.MutatingWebhookConfiguration, name: (uuid: string): string => uuid },
+          { kind: kind.ValidatingWebhookConfiguration, name: (uuid: string): string => uuid },
+        ];
+
+        it.each(
+          sources.flatMap(src =>
+            resourceKinds.map(r => ({
+              source: src.source,
+              getResources: src.getResources,
+              resourceKind: r.kind,
+              name: r.name,
+            })),
+          ),
+        )(
+          "$resourceKind in $source resources should be defined",
+          ({ getResources, resourceKind, name }) => {
+            const res = resource.select(getResources(), resourceKind, name(peprUuid));
             expect(res).toBeDefined();
-          }
-        });
+          },
+        );
 
+        // --- ClusterRoleBinding tests ---
         it.each(sources)(
           "ClusterRoleBinding in $source resources should have correct roleRef and subjects",
           ({ getResources }) => {
-            const clusterRoleBinding = resource.select(
-              getResources(),
-              kind.ClusterRoleBinding,
-              peprUuid,
-            );
-            expect(clusterRoleBinding).toBeDefined();
-            expect(clusterRoleBinding.roleRef.name).toBe(peprUuid);
-            expect(clusterRoleBinding.subjects?.at(0)?.name).toBe(peprUuid);
+            const crb = resource.select(getResources(), kind.ClusterRoleBinding, peprUuid);
+            expect(crb).toBeDefined();
+            expect(crb.roleRef.name).toBe(peprUuid);
+            expect(crb.subjects?.[0]?.name).toBe(peprUuid);
           },
         );
 
+        // --- RoleBinding tests ---
```

Comment:
```suggestion
```

This is already scoped within a `describe` block for `uuid`. The test name makes the purpose of the test clear.